test/e2e/csv_e2e_test.go

```diff
@@ -2664,7 +2664,136 @@ var _ = Describe("CSV", func() {
 		err = waitForDeploymentToDelete(GinkgoT(), c, strategy.DeploymentSpecs[0].Name)
 		require.NoError(GinkgoT(), err)
 	})
+	It("update deployment spec in an existing CSV for a hotfix", func() {
```

Comment:
It looks like you may need to rebase against master. When I look at the raw view of this file, that testify/require package is present in the list of package imports, but not present in this same file in master: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/csv_e2e_test.go#L3-L35, which explains why the e2e tests are failing due `test/e2e/csv_e2e_test.go:3045:3: undefined: require`.

Reply:
rebased the PR with latest on master

Reply:
I verified that the test passes on my local machine.